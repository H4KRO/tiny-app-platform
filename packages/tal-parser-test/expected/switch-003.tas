switch {
  expr1 => value1
  expr2 => value2
  _     => defaultValue
}
