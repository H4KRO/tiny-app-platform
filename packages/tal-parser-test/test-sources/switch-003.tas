switch {
    expr1 => value1
    expr2 => value2
    *     => defaultValue
}