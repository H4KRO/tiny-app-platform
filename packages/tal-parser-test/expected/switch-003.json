[
  {
    "kind": "Switch",
    "value": null,
    "branches": [
      {
        "kind": "SwitchBranch",
        "comparator": {
          "kind": "Local",
          "name": "expr1"
        },
        "value": {
          "kind": "Local",
          "name": "value1"
        }
      },
      {
        "kind": "SwitchBranch",
        "comparator": {
          "kind": "Local",
          "name": "expr2"
        },
        "value": {
          "kind": "Local",
          "name": "value2"
        }
      }
    ],
    "defaultBranch": {
      "value": {
        "kind": "Local",
        "name": "defaultValue"
      }
    }
  }
]